community/libxkbcommon/depends

```diff
@@ -0,0 +1,2 @@
+meson   make
+samurai make
```

Comment:
This can be removed. `meson` pulls in `samurai` automatically.

Reply:
👍